From 4c9f13419d40589499bb51bb9ca9c4358a0db30e Mon Sep 17 00:00:00 2001
From: Zhang Ze <zhangze@rvcore.com>
Date: Wed, 23 Aug 2023 12:19:41 +0800
Subject: [PATCH] add riscv cross compile configs for spdk

---
 dpdkbuild/Makefile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/dpdkbuild/Makefile b/dpdkbuild/Makefile
index 62b7891191..bcb91d40fc 100644
--- a/dpdkbuild/Makefile
+++ b/dpdkbuild/Makefile
@@ -95,6 +95,9 @@ DPDK_OPTS += -Dcpu_instruction_set=$(TARGET_ARCHITECTURE)
 ifneq ($(CONFIG_CROSS_PREFIX),)
 ifeq ($(findstring mingw,$(CONFIG_CROSS_PREFIX)),mingw)
 DPDK_OPTS += --cross-file $(SPDK_ROOT_DIR)/dpdk/config/x86/cross-mingw
+else ifeq ($(findstring riscv64,$(CONFIG_CROSS_PREFIX)),riscv64)
+DPDK_OPTS += --cross-file $(SPDK_ROOT_DIR)/dpdk/config/riscv/meson.cross
+DPDK_OPTS += --native-file $(SPDK_ROOT_DIR)/dpdk/config/riscv/meson.native
 else
 $(error Automatic DPDK cross build is not supported. Please compile DPDK manually \
 with e.g. `meson build --cross-file config/arm/arm64_armv8_linux_gcc`)
-- 
2.34.1

