From f08132a9f56afce2d09326740428a767496f32e7 Mon Sep 17 00:00:00 2001
From: Zhang Ze <zhangze@rvcore.com>
Date: Mon, 14 Aug 2023 15:59:41 +0800
Subject: [PATCH] enable riscv build for yocto

---
 config/riscv/meson.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config/riscv/meson.build b/config/riscv/meson.build
index 07d7d9da23..d3f1d572bc 100644
--- a/config/riscv/meson.build
+++ b/config/riscv/meson.build
@@ -84,8 +84,8 @@ if not meson.is_cross_build()
     endif
 else
     # cross build
-    vendor_id = meson.get_cross_property('vendor_id')
-    arch_id = meson.get_cross_property('arch_id')
+    vendor_id = meson.get_cross_property('vendor_id', 'generic')
+    arch_id = meson.get_cross_property('arch_id', 'generic')
 endif
 
 if not vendors.has_key(vendor_id)
